### PR TITLE
Bare minimum Probot update

### PR DESCRIPTION
**Pull request by LeoColomb**
Moving to v3.0.0 without touching a bit of libs files.

#challengeAccepted ;)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,14 +2,10 @@ const Configuration = require('./lib/configuration')
...
-  async function receive (event) {
-    if (event.payload.repository) {
-      const github = await robot.auth(event.payload.installation.id)
-      const context = new Context(github, event)
-      const config = await Configuration.load(context, '.probot.js')
-      return config.execute()
-    }
-  }
+  robot.on('*', async context => {
+    const github = await robot.auth(context.event.payload.installation.id)
```
Comment (bkeepers) [inline index.js]: This can be replaced with `context.github`

File: index.js [hunk 2]
```diff
@@ -2,14 +2,10 @@ const Configuration = require('./lib/configuration')
...
+    const contextOld = new Context(github, context.event)
```
Comment (bkeepers) [inline index.js]: I don't think `context.event` is right here since that's only the event name and not the payload.

File: index.js (hunk 1 above)
Comment (LeoColomb) [reply index.js]: It should be the same, isn't?
Anyway, fixed.

File: index.js (hunk 2 above)
Comment (LeoColomb) [reply index.js]: Good catch! Dumb mistake. Fixed!

**Comment by LeoColomb** [comment]
@bkeepers Up! 😇

**Comment by bkeepers** [comment]
Looks great, thanks!

**Pull request merged**